modules/serverless-concurrency-limits-configure-hard.adoc

```diff
@@ -24,9 +24,9 @@ spec:
       containerConcurrency: 50
```

Comment:
Same as above. If time permits, please comment which assemblies this module is included in.